### PR TITLE
Add codespaces config

### PR DESCRIPTION
**Pull request by benjaminwood**
The intent here is to add config for codespaces that does not break our existing docker-compose dev env setup.

Seems to work! :smile:

### CODE REVIEW

File: docker/ruby/Dockerfile [hunk 1]
```diff
@@ -35,7 +35,6 @@ RUN groupadd -g $GID $USER && \
 ENV LANG C.UTF-8
 
 # Point Bundler at /gems. This will cause Bundler to re-use gems that have already been installed on the gems volume
-ENV BUNDLE_PATH /gems
 ENV BUNDLE_HOME /gems
```
Comment (nvick) [inline docker/ruby/Dockerfile:38]: BUNDLE_HOME is no longer a Bundler config environment variable. This could be cleaned up now too.

**Comment by nvick** [comment]
The entrypoint needs an update to the updated working directory. 

https://github.com/OfficeMomsandDads/scheduler/blob/23388a0ac65f152949eab220bf3f307ae9cd6d0e/docker/rails-entrypoint#L4

**Comment by benjaminwood** [comment]
Thanks!

Ready for re-review. I'm also building from scratch both locally & in codespaces, just to ensure everything is working in both contexts one last time.

**Review by nvick — APPROVED** [review]
Awesome!

**Pull request merged**